redisinsight/api/src/modules/browser/dto/z-set.dto.ts

```diff
@@ -77,13 +77,12 @@ export class ZSetMemberDto {
 
   @ApiProperty({
     description: 'Member score value.',
-    type: Number,
+    type: Number || String,
     default: 1,
   })
   @IsDefined()
-  @IsNumber({ maxDecimalPlaces: 15 })
-  @Type(() => Number)
-  score: number;
+  @isZSetScore()
+  score: number | 'inf' | '-inf';
```

Comment:
It is better to create a type (like `type ZSetScore = ...`) and use it everywhere